app/api/tickets.py

```diff
@@ -31,6 +32,10 @@ def before_post(self, args, kwargs, data):
             raise ObjectNotFound({'parameter': 'event_id'},
                                  "Event: {} not found".format(data['event_id']))
 
+        query = db.session.query(Ticket.id).filter_by(name=data['name'])
```

Comment:
You could use the `get_count` helper at https://github.com/fossasia/open-event-server/blob/development/app/api/helpers/db.py#L69 instead of this

Reply:
Ya that will be better.